### ISSUE TITLE
[tfjs-react-native] Android app terminating during large model load

### ISSUE THREAD

**Issue by burgreen** [tensorflow/tfjs#3600]
#### Browser version: Android mobile version 9

I am customizing the integration_rn59 example to use my own pre-trained models being read in via bundleResourceIO.

The json and binary weights files for my model is 0.5Mb and 217Mb, respectively. 

The app is quietly terminating during model loading. I have 43Gb of available storage on my device, but I realize the RAM is probably on the order of 2-4Gb. I have gotten one downloaded MobileNet model to run and inference without any memory issues (accuracy is still on the suspect side). 

My questions are: How large of model can be loaded and used in practice on a mobile device? Am I asking my device to do something unreasonable by using such a large custom model? I am doing lots of console.log debugging, and all is well until model loading, which consistently quietly terminates the app.

**Comment by burgreen** [comment]
Yep, that model is too large for my phone RAM.
